src/newgrf_gui.cpp

```diff
@@ -654,7 +656,7 @@ struct NewGRFWindow : public Window, NewGRFScanCallback {
 		this->avails.SetFilterFuncs(this->filter_funcs);
 		this->avails.ForceRebuild();
 
-		this->OnInvalidateData(GOID_NEWGRF_LIST_EDITED);
+		this->InvalidateData(GOID_NEWGRF_LIST_EDITED);
```

Comment:
Does this change do anything? A newly created window will be fully redrawn anyway, so why use `InvalidateData`?

Reply:
It probably doesn't, I  just thought it was not technically correct to make a direct call to a callback function rather than triggering the callback from InvalidateData. I can revert, thanks for the comment!